In [ ]:
!apt-get install tesseract-ocr -y
!pip install pytesseract pillow

import pytesseract
from PIL import Image

img = Image.open('gpt papers sequence.png')
text = pytesseract.image_to_string(img)

print(text)

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
tesseract-ocr is already the newest version (4.1.1-2.1build1).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
PJTiny tia
PN a a ese e- 1B Y(ele]
need (Vaswani et.al).

Birth of attention
INT Te antl lay
translation by jointly
learning to align and
translate

Just ask
Language models are
few shot learners
(Brown et.al.)

Generalize
Improving language
understanding by
Generative pre-training
(Radford et.al)

 

PBL
Bcc a Claret) =)
models to follow
instructions.

Use tools
Toolformer: Language
Models can teach
themselves to use tools
(Schick, Yu)

BTor CR CRT Tiler)
Ke) Tae eatLtLe
Adaptation of Large
Language Models (Hu,
Shen, et.al.)



# Task 2: Learning Material Creation

In [ ]:
## OCR Method

The provided PDF was image-based. Text extraction was performed using:

- Google Docs OCR (primary method for better accuracy)
- Tesseract OCR in Google Colab (for experimentation)

The extracted content was cleaned and structured before generating learning materials.

## Extracted Content (Preview)

task 2




n
Chapter Eight
12089CH08
ELECTROMAGNETIC
WAVES
NCERT
repablished
8.1 INTRODUCTION
In Chapter 4, we learnt that an electric current produces magnetic field and that two current-carrying wires exert a magnetic force on each other. Further, in Chapter 6, we have seen that a magnetic field changing with time gives rise to an electric field. Is the converse also true? Does an electric field changing with time give rise to a magnetic field? James Clerk Maxwell (1831-1879), argued that this was indeed the case - not only an electric current but also a time-varying electric field generates magnetic field. While applying the Ampere's circuital law to find magnetic field at a point outside a capacitor connected to a time-varying current, Maxwell noticed an inconsistency in the Ampere's circuital law. He suggested the existence of an additional current, called by him, the displacement current to remove this inconsistency.
Maxwell formulated a set of equations involving electric and magnetic fields, and their sources, the charge and current densities. These equations are known as Maxwell's equations. Together with the Lorentz force formula (Chapter 4), they mathematically express all the basic laws of electromagnetism.
The most important prediction to emerge from Maxwell's equations is the existence of electromagnetic waves, which are (coupled) time- varying electric and magnetic fields that propagate in space. The speed of the waves, according to these equations, turned out to be very close to
Reprint 2026-27
JAMES CLERK MAXWELL (1831-1879)
Physics
James Clerk Maxwell (1831-1879) Born in Edinburgh, Scotland, was among the greatest physicists of the nineteenth century. He derived the thermal velocity distribution of molecules in a gas and was among the first to obtain
reliable estimates of molecular parameters from measurable quantities like viscosity, etc. Maxwell's greatest acheivement was the unification of the laws of electricity
and magnetism (discovered by Coulomb, Oersted, Ampere and Faraday) into a consistent set of equations now called Maxwell's equations. From these he arrived at the most important conclusion that light is an electromagnetic wave. Interestingly, Maxwell did not agree with the idea (strongly suggested by the Faraday's laws electrolysis)
electricity
of
that
was
particulate in nature.
202
the speed of light( 3 ×108 m/s), obtained from optical measurements. This led to the remarkable conclusion that light is an electromagnetic wave. Maxwell's work thus unified the domain of electricity, magnetism and light. Hertz, in 1885, experimentally demonstrated the existence of electromagnetic waves. Its technological use by Marconi and others led in due course to the revolution in communication that we are witnessing today.
In this chapter, we first discuss the need for displacement current and its consequences. Then we present a descriptive account of electromagnetic waves. The broad spectrum of electromagnetic waves, stretching from y rays (wavelength ~10-12 m) to long radio waves (wavelength ~106 m) is described.
8.2 DISPLACEMENT CURRENT
We have seen in Chapter 4 that an electrical current produces a magnetic field around it. Maxwell showed that for logical consistency, a changing electric field must also produce a magnetic field. This effect is of great importance because it explains the existence of radio waves, gamma rays and visible light, as well as all other forms of electromagnetic waves.
To see how a changing electric field gives rise to a magnetic field, let us consider the process of charging of a capacitor and apply Ampere's circuital law given by (Chapter 4)
$B.dl=
μo i (t)
(8.1)
to find magnetic field at a point outside the capacitor. Figure 8.1(a) shows a parallel plate capacitor C which is a part of circuit through which a time-dependent current i (t) flows. Let us find the magnetic field at a point such as P, in a region outside the parallel plate capacitor. For this, we consider a plane circular loop of radius r whose plane is perpendicular to the direction of the current-carrying wire, and which is centred symmetrically with respect to the wire [Fig. 8.1(a)]. From symmetry, the magnetic field is directed along the circumference of the circular loop and is the same in magnitude at all points on the loop so that if B is the magnitude of the field, the left side of Eq. (8.1) is B (2π г). So we have
В (2πг) = μi(t)
(8.2)
Reprint 2026-27
Electromagnetic Waves
Now, consider a different surface, which has the same boundary. This is a pot like surface [Fig. 8.1(b)] which nowhere touches the current, but has its bottom between the capacitor plates; its mouth is the circular loop mentioned above. Another such surface is shaped like a tiffin box (without the lid) [Fig. 8.1(c)]. On applying Ampere's circuital law to such surfaces with the same perimeter, we find that the left hand side of Eq. (8.1) has not changed but the right hand side is zero and not μi, since no current passes through the surface of Fig. 8.1(b) and (c). So we have a contradiction; calculated one way, there is a magnetic field at a point P; calculated another way, the magnetic field at P is zero. Since the contradiction arises from our use of Ampere's circuital law, this law must be missing something. The missing term must be such that one gets the same magnetic field at point P, no matter what surface is used.
i(t)
i(t)
P
+ +o+ + + + + + +
Σ
(a)
C
+ + + + +++++
she
US
We can actually guess the missing term by looking carefully at Fig. 8.1(c). Is there anything passing through the surface S between the plates of the capacitor? Yes, of course, the electric field! If the plates of the capacitor have an area A, and a total charge Q, the magnitude of the electric field E between the plates is (Q/A)/ (see Eq. 2.41). The field is perpendicular to the surface S of Fig. 8.1(c). It has the same magnitude over the area A of the capacitor plates, and vanishes outside it. So what
is the electric flux 0 through the surface S? Using Gauss's law, it i
E
=EA =
1 Q Το Α
Q
A
=
Now if the charge
(8.3)
on the capacitor plates changes with time, there is a
current i = (dQ/dt), so that using Eq. (8.3), we have
dPE
=
d Q 1 do
=
dt dt o Єo dt
This implies that for consistency,
be
i(t)
C
C
1
(b)
S
Το
(dd)
= i
dt
(8.4)
i
This is the missing term in Ampere's circuital law. If we generalise this law by adding to the total current carried by conductors through the surface, another term which is & times the rate of change of electric flux through the same surface, the total has the same value of current for all surfaces. If this is done, there is no contradiction in the value of B obtained anywhere using the generalised Ampere's law. B at the point P is non-zero no matter which surface is used for calculating it. B at a point P outside the plates [Fig. 8.1(a)] is the same as at a point M just inside, as it should be. The current carried by conductors due to flow of charges is called conduction current. The current, given by Eq. (8.4), is a new term, and is due to changing electric field (or electric displacement, an old term still used sometimes). It is, therefore, called displacement current or Maxwell's displacement current. Figure 8.2 shows the electric and magnetic fields inside the parallel plate capacitor discussed above.
The generalisation made by Maxwell then is the following. The source of a magnetic field is not just the conduction electric current due to flowing
(c)
FIGURE 8.1 A parallel plate capacitor C, as part of a circuit through which a time dependent current
i (t) flows, (a) a loop of radius r, to determine magnetic field at a point P on the loop; (b) a pot-shaped surface passing through the interior between the capacitor plates with the loop shown in (a) as its rim; (c) a tiffin- shaped surface with the circular loop as its rim and a flat circular bottom S
between the capacitor
plates. The arrows show uniform electric
field between the capacitor plates.
203
Reprint 2026-27
i(t)
B
X m
X
Po
X
X
X
Physics
M
...
C
(a)
B
(b)
A
E
B
+
X
R
X
X
X
E
B
FIGURE 8.2 (a) The electric and magnetic fields E and B between the capacitor plates, at the point M. (b) A cross sectional view of Fig. (a).
204
charges, but also the time rate of change of electric field. More precisely, the total current i is the sum of the conduction current denoted by i, and the displacement current denoted by i, (= (do/dt)). So we have
i=i ̧ +i¿=¿¿ + εo
dPE dt
(8.5)
In explicit terms, this means that outside the capacitor plates, we have only conduction current i = i, and no displacement current, i.e., i=0. On the other hand, inside the capacitor, there is no conduction current, i.e., i̟ = 0, and there is only displacement current, so that i̟¿ = i.
The generalised (and correct) Ampere's circuital law has the same form as Eq. (8.1), with one difference: "the total current passing through any surface of which the closed loop is the perimeter" is the sum of the conduction current and the displacement current. The generalised law is
$B•dl= μo ic +μo εo
dPE dt
Hist
and is known as Ampere-Maxwell law.
(8.6)
In all respects, the displacement current has the same physical effects as the conduction current. In some cases, for example, steady electric fields in a conducting wire, the displacement current may be zero since the electric field E does not change with time. In other cases, for example, the charging capacitor above, both conduction and displacement currents may be present in different regions of space. In most of the cases, they both may be present in the same region of space, as there exist no perfectly conducting or perfectly insulating medium. Most interestingly, there may be large regions of space where there is no conduction current, but there is only a displacement current due to time-varying electric fields. In such a region, we expect a magnetic field, though there is no (conduction) current source nearby! The prediction of such a displacement current can be verified experimentally. For example, a magnetic field (say at point M) between the plates of the capacitor in Fig. 8.2(a) can be measured and is seen to be the same as that just outside (at P).
not
The displacement current has (literally) far reaching consequences. One thing we immediately notice is that the laws of electricity and magnetism are now more symmetrical*. Faraday's law of induction states that there is an induced emf equal to the rate of change of magnetic flux. Now, since the emf between two points 1 and 2 is the work done per unit charge in taking it from 1 to 2, the existence of an emf implies the existence of an electric field. So, we can rephrase Faraday's law of electromagnetic induction by saying that a magnetic field, changing with time, gives rise to an electric field. Then, the fact that an electric field changing with time gives rise to a magnetic field, is the symmetrical counterpart, and is
They are still not perfectly symmetrical; there are no known sources of magnetic field (magnetic monopoles) analogous to electric charges which are sources of electric field.
Reprint 2026-27
Electromagnetic Waves
a consequence of the displacement current being a source of a magnetic field. Thus, time-dependent electric and magnetic fields give rise to each other! Faraday's law of electromagnetic induction and Ampere-Maxwell law give a quantitative expression of this statement, with the current being the total current, as in Eq. (8.5). One very important consequence of this symmetry is the existence of electromagnetic waves, which we discuss qualitatively in the next section.
MAXWELL'S EQUATIONS IN VACUUM
1. §E.dA = Q/ε0
2. §B.dA = 0
3. §E.dl = ¬dø
d1
-dPB dt
4. §B.dl=μoic + μoεo
(Gauss's Law for electricity)
(Gauss's Law for magnetism)
(Faraday's Law)
E
Qublished
dPE dt
(Ampere - Maxwell Law)
8.3 ELECTROMAGNETIC WAVES
8.3.1 Sources of electromagnetic waves
How are electromagnetic waves produced? Neither stationary charges nor charges in uniform motion (steady currents) can be sources of electromagnetic waves. The former produces only electrostatic fields, while the latter produces magnetic fields that, however, do not vary with time. It is an important result of Maxwell's theory that accelerated charges radiate electromagnetic waves. The proof of this basic result is beyond the scope of this book, but we can accept it on the basis of rough, qualitative reasoning. Consider a charge oscillating with some frequency. (An oscillating charge is an example of accelerating charge.) This produces an oscillating electric field in space, which produces an oscillating magnetic field, which in turn, is a source of oscillating electric field, and so on. The oscillating electric and magnetic fields thus regenerate each other, so to speak, as the wave propagates through the space. The frequency of the electromagnetic wave naturally equals the frequency of oscillation of the charge. The energy associated with the propagating wave comes at the expense of the energy of the source – the accelerated charge.
From the preceding discussion, it might appear easy to test the prediction that light is an electromagnetic wave. We might think that all we needed to do was to set up an ac circuit in which the current oscillate at the frequency of visible light, say, yellow light. But, alas, that is not possible. The frequency of yellow light is about 6 × 1014 Hz, while the frequency that we get even with modern electronic circuits is hardly about 1011 Hz. This is why the experimental demonstration of electromagnetic
205
Reprint 2026-27
Physics
Heinrich Rudolf Hertz
-
(1857 1894) German physicist who was the
first to broadcast and receive radio waves. He produced electro-
magnetic waves, sent them through space, and measured their wave- length and speed. He showed that the nature of their vibration, reflection and refraction was the same as that of light and heat waves, establishing their identity for the first time. He also pioneered research on discharge of electricity through gases, and discovered the
photoelectric effect.
X
E
ot
wave had to come in the low frequency region (the radio wave region), as in the Hertz's experiment (1887).
Hertz's successful experimental test of Maxwell's theory created a sensation and sparked off other important works in this field. Two important achievements in this connection deserve mention. Seven years after Hertz, Jagdish Chandra Bose, working at Calcutta (now Kolkata), succeeded in producing and observing electromagnetic waves of much shorter wavelength (25 mm to 5 mm). His experiment, like that of Hertz's, was confined to the laboratory.
At around the same time, Guglielmo Marconi in Italy followed Hertz's work and succeeded in transmitting electromagnetic waves over distances of many kilometres. Marconi's experiment marks the beginning of the field of communication using electromagnetic waves.
8.3.2 Nature of electromagnetic waves
It can be shown from Maxwell's equations that electric and magnetic fields in an electromagnetic wave are perpendicular to each other, and to the direction of propagation. It appears reasonable, say from our discussion of the displacement current. Consider Fig. 8.2. The electric field inside the plates of the capacitor is directed perpendicular to the plates. The magnetic field this gives rise to via the displacement current is along the perimeter of a circle parallel to the capacitor plates. So B and E are perpendicular in this case. This is a general feature.
Χ
In Fig. 8.3, we show a typical example of a plane electromagnetic wave propagating along the z direction (the fields are shown as a function of the z coordinate, at a given time t). The electric field E is along the x-axis, and varies sinusoidally with z, at a given time. The magnetic field B1, is along the y-axis, and again varies sinusoidally with z. The electric and magnetic fields E
and B are perpendicular to each other, and to the direction z of
y
y
x
x
propagation. We can write E and Bas follows:
HEINRICH RUDOLF HERTZ (1857-1894)
E= E sin (kz-wt)
[8.7(a)]
B=B sin (kz-ot)
[8.7(b)]
y
B
y
FIGURE 8.3 A linearly polarised electromagnetic wave, propagating in the z-direction with the oscillating electric field E along the x-direction and the oscillating magnetic field B along the y-direction.
Here k is related to the wave length
2 of the wave by the usual equation
k
=
2元
λ
(8.8)
206
Reprint 2026-27
Electromagnetic Waves
and @ is the angular frequency. k is the magnitude of the wave vector (or propagation vector) k and its direction describes the direction of propagation of the wave. The speed of propagation of the wave is (@/k). Using Eqs. [8.7(a) and (b)] for E and B and Maxwell's equations, one finds that
@=
ck, where, c = 1/√√μ。。
X
y
[8.9(a)]
The relation = ck is the standard one for waves (see for example, Section 14.4 of class XI Physics textbook). This relation is often written in terms of frequency, v (=o/2n) and wavelength, λ (=2π/k) as
2πν = c
2π
λ
or
shed
vλ = c
[8.9(b)]
It is also seen from Maxwell's equations that the magnitude of the electric and the magnetic fields in an electromagnetic wave are related as
Bo=(E/c)
(8.10)
We here make remarks on some features of electromagnetic waves. They are self-sustaining oscillations of electric and magnetic fields in free space, or vacuum. They differ from all the other waves we have studied so far, in respect that no material medium is involved in the vibrations of the electric and magnetic fields.
But what if a material medium is actually there? We know that light, an electromagnetic wave, does propagate through glass, for example. We have seen earlier that the total electric and magnetic fields inside a medium are described in terms of a permittivity & and a magnetic permeability μ (these describe the factors by which the total fields differ from the external fields). These replace & and 4 in the description to
ဝ
мо electric and magnetic fields in Maxwell's equations with the result that in a material medium of permittivity & and magnetic permeability μ, the velocity of light becomes,
V =
1
με
b
(8.11)
Thus, the velocity of light depends on electric and magnetic properties of the medium. We shall see in the next chapter that the refractive index of one medium with respect to the other is equal to the ratio of velocities of light in the two media.
The velocity of electromagnetic waves in free space or vacuum is an important fundamental constant. It has been shown by experiments on electromagnetic waves of different wavelengths that this velocity is the same (independent of wavelength) to within a few metres per second, out of a value of 3×108 m/s. The constancy of the velocity of em waves in vacuum is so strongly supported by experiments and the actual value is so well known now that this is used to define a standard of length.
The great technological importance of electromagnetic waves stems from their capability to carry energy from one place to another. The radio and TV signals from broadcasting stations carry energy. Light carries energy from the sun to the earth, thus making life possible on the earth.
Reprint 2026-27
207
Physics
Example 8.1 A plane electromagnetic wave of frequency 25 MHz travels in free space along the x-direction. At a particular point in space and time, E = 6.3 j V/m. What is B at this point?
Solution Using Eq. (8.10), the magnitude of B is
http://www.fnal.gov/pub/inquiring/more/light http://imagine.gsfc.nasa.gov/docs/science/
Electromagnetic spectrum
PHYSICS
208
EXAMPLE 8.1
EXAMPLE 8.2
E
B=
=
6.3 V/m 3x108 m/s
=2.1x10-8 T
To find the direction, we note that E is along y-direction and the wave propagates along x-axis. Therefore, B should be in a direction perpendicular to both x- and y-axes. Using vector algebra, E x B should be along x-direction. Since, (+j) × (+ k) = i, B is along the z-direction. Thus, B = 2.1 × 10-8 kT
Example 8.2 The magnetic field in a plane electromagnetic wave is given by B = (2 × 10-7) T sin (0.5×103x+1.5×1011t).
y
(a) What is the wavelength and frequency of the wave?
(b) Write an expression for the electric field.
Solution
(a) Comparing the given equation with
Χ
t
B=B Sin 2p
Sin (2P(++)
y
λ
T
We get, λ =
2π 0.5×103
m = 1.26 cm,
Candy=(1.5x10")/2x=23.9 GHz
T
(b) E = Boc= 2×10-7 T × 3 × 108 m/s = 6 × 101 V/m
The electric field component is perpendicular to the direction of propagation and the direction of magnetic field. Therefore, the electric field component along the z-axis is obtained as
E = 60 sin (0.5 × 103x + 1.5 × 1011 t) V/m
Z
8.4 ELECTROMAGNETIC SPECTRUM
8.4
not
At the time Maxwell predicted the existence of electromagnetic waves, the only familiar electromagnetic waves were the visible light waves. The existence of ultraviolet and infrared waves was barely established. By the end of the nineteenth century, X-rays and gamma rays had also been discovered. We now know that, electromagnetic waves include visible light waves, X-rays, gamma rays, radio waves, microwaves, ultraviolet and infrared waves. The classification of em waves according to frequency is the electromagnetic spectrum (Fig. 8.4). There is no sharp division between one kind of wave and the next. The classification is based roughly on how the waves are produced and/or detected.
We briefly describe these different types of electromagnetic waves, in order of decreasing wavelengths.
Reprint 2026-27
Electromagnetic
Waves
Frequency, Hz
Wavelength, m
1023
1022
10-14
1021
Gamma rays
10-13
400 nm
1020
- 10-12
Violet
1019
10-11
1018
X-rays
10-10
450 nm.
1017
10%
1016
Ultraviolet
10-8
1015
107
500 nm
1014
Visible
10-6
1013
Infrared.
10-5
1012
104
550 nm
1011
Microwaves
10
1010
10
109
Short radio waves
10
600 nm
108
-Television and FM radio
107
{
1
106
AM radio
105
103
101
102
650 nm
104
Long radio waves.
104
104
105
102
106
10
107
心
p
ublished
Blue
Green
Yellow
FIGURE 8.4 The electromagnetic spectrum, with common names for various part of it. The various regions do not have sharply defined boundaries.
8.4.1 Radio waves
Radio waves are produced by the accelerated motion of charges in conducting wires. They are used in radio and television communication systems. They are generally in the frequency range from 500 kHz to about 1000 MHz. The AM (amplitude modulated) band is from 530 kHz to 1710 kHz. Higher frequencies upto 54 MHz are used for short wave bands. TV waves range from 54 MHz to 890 MHz. The FM (frequency modulated) radio band extends from 88 MHz to 108 MHz. Cellular phones use radio waves to transmit voice communication in the ultrahigh frequency (UHF) band. How these waves are transmitted and received is described in Chapter 15.
8.4.2 Microwaves
Microwaves (short-wavelength radio waves), with frequencies in the gigahertz (GHz) range, are produced by special vacuum tubes (called klystrons, magnetrons and Gunn diodes). Due to their short wavelengths, they are suitable for the radar systems used in aircraft navigation. Radar also provides the basis for the speed guns used to time fast balls, tennis- serves, and automobiles. Microwave ovens are an interesting domestic application of these waves. In such ovens, the frequency of the microwaves is selected to match the resonant frequency of water molecules so that energy from the waves is transferred efficiently to the kinetic energy of the molecules. This raises the temperature of any food containing water.
Reprint 2026-27
Microwave
oven
444-446-41
209
210
Physics
8.4.3 Infrared waves
Infrared waves are produced by hot bodies and molecules. This band lies adjacent to the low-frequency or long-wave length end of the visible spectrum. Infrared waves are sometimes referred to as heat waves. This is because water molecules present in most materials readily absorb infrared waves (many other molecules, for example, CO2, NH,, also absorb infrared waves). After absorption, their thermal motion increases, that is, they heat up and heat their surroundings. Infrared lamps are used in physical therapy. Infrared radiation also plays an important role in maintaining the earth's warmth or average temperature through the greenhouse effect. Incoming visible light (which passes relatively easily through the atmosphere) is absorbed by the earth's surface and re- radiated as infrared (longer wavelength) radiations. This radiation is trapped by greenhouse gases such as carbon dioxide and water vapour. Infrared detectors are used in Earth satellites, both for military purposes and to observe growth of crops. Electronic devices (for example semiconductor light emitting diodes) also emit infrared and are widely used in the remote switches of household electronic systems such as TV sets, video recorders and hi-fi systems.
8.4.4 Visible rays
It is the most familiar form of electromagnetic waves. It is the part of the spectrum that is detected by the human eye. It runs from about 4 x 1014 Hz to about 7 x 1014 Hz or a wavelength range of about 700- 400 nm. Visible light emitted or reflected from objects around us provides us information about the world. Our eyes are sensitive to this range of wavelengths. Different animals are sensitive to different range of wavelengths. For example, snakes can detect infrared waves, and the 'visible' range of many insects extends well into the utraviolet.
8.4.5 Ultraviolet rays
It covers wavelengths ranging from about 4 × 107 m (400 nm) down to 6 × 10-10m (0.6 nm). Ultraviolet (UV) radiation is produced by special lamps and very hot bodies. The sun is an important source of ultraviolet light. But fortunately, most of it is absorbed in the ozone layer in the atmosphere at an altitude of about 40-50 km. UV light in large quantities has harmful effects on humans. Exposure to UV radiation induces the production of more melanin, causing tanning of the skin. UV radiation is absorbed by ordinary glass. Hence, one cannot get tanned or sunburn through glass windows.
not
Welders wear special glass goggles or face masks with glass windows to protect their eyes from large amount of UV produced by welding arcs. Due to its shorter wavelengths, UV radiations can be focussed into very narrow beams for high precision applications such as LASIK (Laser- assisted in situ keratomileusis) eye surgery. UV lamps are used to kill germs in water purifiers.
Ozone layer in the atmosphere plays a protective role, and hence its depletion by chlorofluorocarbons (CFCs) gas (such as freon) is a matter of international concern.
Reprint 2026-27
Electromagnetic Waves
8.4.6 X-rays
Beyond the UV region of the electromagnetic spectrum lies the X-ray region. We are familiar with X-rays because of its medical applications. It covers wavelengths from about 10-8 m (10 nm) down to 10-13 (10 nm). One common way to generate X-rays is to bombard a metal target by high energy electrons. X-rays are used as a diagnostic tool in medicine and as a treatment for certain forms of cancer. Because X-rays damage or destroy living tissues and organisms, care must be taken to avoid unnecessary or over exposure.
8.4.7 Gamma rays
They lie in the upper frequency range of the electromagnetic spectrum and have wavelengths of from about 10-10m to less than 10-14m. This high frequency radiation is produced in nuclear reactions and also emitted by radioactive nuclei. They are used in medicine to destroy cancer cells.
shed
Table 8.1 summarises different types of electromagnetic waves, their production and detections. As mentioned earlier, the demarcation between different regions is not sharp and there are overlaps.
TABLE 8.1 DIFFERENT TYPES OF ELECTROMAGNETIC WAVES
Wavelength range
Туре
> 0.1 m
Radio
Microwave
0.1m to 1 mm 1 mm
Infra-red
1mm to 700 nm
Production
Rapid acceleration and decelerations of electrons
in aerials
Klystron valve or
magnetron valve
Detection
Receiver's aerials
Point contact diodes
Light
700 nm to 400 nm
Ultraviolet
X-rays
400 nm to 1nm
n
1nm to 10-3 nm
<10-3 nm
Gamma rays
Vibration of atoms
and molecules
Electrons in atoms emit light when they move from one energy level to a lower energy level
Inner shell electrons in atoms moving from one energy level to a lower level
X-ray tubes or inner shell electrons
Radioactive decay of the nucleus
Thermopiles
Bolometer, Infrared photographic film
The eye Photocells
Photographic film
Photocells
Photographic film
Photographic film Geiger tubes Ionisation chamber
-do-
Reprint 2026-27
211
Physics
212
SUMMARY
1. Maxwell found an inconsistency in the Ampere's law and suggested the existence of an additional current, called displacement current, to remove this inconsistency. This displacement current is due to time-varying electric field and is given by
id = Ɛo
d
dt
and acts as a source of magnetic field in exactly the same way as conduction current.
2. An accelerating charge produces electromagnetic waves. An electric charge oscillating harmonically with frequency v, produces electromagnetic waves of the same frequency v. An electric dipole is a basic source of electromagnetic waves.
3. Electromagnetic waves with wavelength of the order of a few metres were first produced and detected in the laboratory by Hertz in 1887. He thus verified a basic prediction of Maxwell's equations.
4. Electric and magnetic fields oscillate sinusoidally in space and time in an electromagnetic wave. The oscillating electric and magnetic fields, E and B are perpendicular to each other, and to the direction of propagation of the electromagnetic wave. For a wave of frequency v, wavelength 2, propagating along z-direction, we have
E = E(t) = E sin (kz - wt)
= E sin
vt
sin
t
[2m(-vn)]= Bo sum[2x(1-1)]
λ
B = B ̧(t) = B ̧ sin (kz – @ t)
=
λ T
B, sin [2-(2 – v )] = B, sin [2-(— ;)]
2л
λ
They are related by E/B = c. They
λ T
(the
5. The speed c of electromagnetic wave in vacuum is related to Мо and
free space permeability and permittivity constants) as follows:
c = 1/√μ。。. The value of c equals the speed of light obtained from
optical measurements.
Light is an electromagnetic wave; c is, therefore, also the speed of light. Electromagnetic waves other than light also have the same velocity c in free space.
The speed of light, or of electromagnetic waves in a material medium is given by v = 1/√με
where μ is the permeability of the medium and & its permittivity.
6. The spectrum of electromagnetic waves stretches, in principle, over an infinite range of wavelengths. Different regions are known by different names; y-rays, X-rays, ultraviolet rays, visible rays, infrared rays, microwaves and radio waves in order of increasing wavelength from 10-2 Å or 10-12 m to 106 m.
They interact with matter via their electric and magnetic fields which set in oscillation charges present in all matter. The detailed interaction and so the mechanism of absorption, scattering, etc., depend on the wavelength of the electromagnetic wave, and the nature of the atoms and molecules in the medium.
Reprint 2026-27
Electromagnetic Waves
POINTS TO PONDER
1. The basic difference between various types of electromagnetic waves lies in their wavelengths or frequencies since all of them travel through vacuum with the same speed. Consequently, the waves differ considerably in their mode of interaction with matter.
2. Accelerated charged particles radiate electromagnetic waves. The wavelength of the electromagnetic wave is often correlated with the characteristic size of the system that radiates. Thus, gamma radiation, having wavelength of 10-14 m to 10-15 m, typically originate from an atomic nucleus. X-rays are emitted from heavy atoms. Radio waves are produced by accelerating electrons in a circuit. A transmitting antenna can most efficiently radiate waves having a wavelength of about the same size as the antenna. Visible radiation emitted by atoms is, however, much longer in wavelength than atomic size.
3. Infrared waves, with frequencies lower than those of visible light, vibrate not only the electrons, but entire atoms or molecules of a substance. This vibration increases the internal energy and consequently, the temperature of the substance. This is why infrared waves are often called heat waves.
4.
N
repolished
The centre of sensitivity of our eyes coincides with the centre of the wavelength distribution of the sun. It is because humans have evolved with visions most sensitive to the strongest wavelengths from the sun.
8.1
8.2
EXERCISES
Figure 8.5 shows a capacitor made of two circular plates each of radius 12 cm, and separated by 5.0 cm. The capacitor is being charged by an external source (not shown in the figure). The charging current is constant and equal to 0.15A.
(a) Calculate the capacitance and the rate of change of potential
difference between the plates.
(b) Obtain the displacement current across the plates.
(c) Is Kirchhoff's first rule (junction rule) valid at each plate of the
capacitor? Explain.
ӨӨ
FIGURE 8.5
A parallel plate capacitor (Fig. 8.6) made of circular plates each of radius R = 6.0 cm has a capacitance C = 100 pF. The capacitor is connected to a 230 V ac supply with a (angular) frequency of 300 rad s1.
Reprint 2026-27
213
214
Physics
(a) What is the rms value of the conduction current?
(b) Is the conduction current equal to the displacement current?
Determine the amplitude of B at a point 3.0 cm from the axis between the plates.
(c)
Ө
8.3
8.4
8.5
8.6
8.7
8.8
8.9
8.10
FIGURE 8.6
What physical quantity is the same for X-rays of wavelength 10-10 m, red light of wavelength 6800 Å and radiowaves of wavelength
500m?
A plane electromagnetic wave travels in vacuum along z-direction.
along What can you say about the directions of its electric and magnetic field vectors? If the frequency of the wave is 30 MHz, what is its wavelength?
A radio can tune in to any station in the 7.5 MHz to 12 MHz band. What is the corresponding wavelength band?
A charged particle oscillates about its mean equilibrium position with a frequency of 109 Hz. What is the frequency of the electromagnetic waves produced by the oscillator?
The amplitude of the magnetic field part of a harmonic electromagnetic wave in vacuum is B = 510 nT. What is the amplitude of the electric field part of the wave? Suppose that the electric field amplitude of an electromagnetic wave is E 120 N/C and that its frequency is v = 50.0 MHz. (a) Determine, B1,o, k, and 2. (b) Find expressions for E and B.
=
The terminology of different parts of the electromagnetic spectrum is given in the text. Use the formula E = hv (for energy of a quantum of radiation: photon) and obtain the photon energy in units of eV for different parts of the electromagnetic spectrum. In what way are the different scales of photon energies that you obtain related to the sources of electromagnetic radiation?
In a plane electromagnetic wave, the electric field oscillates sinusoidally at a frequency of 2.0 × 1010 Hz and amplitude 48 V m1. (a) What is the wavelength of the wave?
(b)
What is the amplitude of the oscillating magnetic field? (c) Show that the average energy density of the E field equals the
average energy density of the B field. [c = 3 x 108 m s1.]
not
Reprint 2026-27





### 🟢 Easy Version (Simplified Electromagnetic Waves )

#### 1. Introduction  
From earlier chapters you already know:  
- Electric current produces a magnetic field.  
- A changing magnetic field produces an electric field (Faraday’s law).  

Now, Maxwell asked:  
👉 **Can a changing electric field produce a magnetic field?**  
Answer: **Yes.**  
This idea is the key to electromagnetic waves. [Based on your extracted NCERT text]

***

#### 2. What Maxwell Added  
Maxwell noticed that Ampere’s law gave different answers for the same magnetic field, depending on which surface you chose near a charging capacitor.  
To fix this, he introduced a new idea:  
👉 **Displacement current** — a kind of “current” due to a **changing electric field** even when no real charges are flowing.  

So the **total current** is:  
\[
\text{Total current} = \text{conduction current} + \text{displacement current}
\]
- Conduction current: due to moving charges in wires.  
- Displacement current: due to changing electric field / changing electric flux. [Based on your text]

***

#### 3. Capacitor Example (Simple Picture)  
Imagine a capacitor being charged:  
- Between the plates, **no charges flow**, so **no conduction current**.  
- But the **electric field between the plates is changing**.  
Still, a **magnetic field** is observed around that region.  

Maxwell said:  
👉 A **changing electric field acts like a current** — this is the displacement current.  
This keeps the magnetic field consistent, no matter which surface you choose in the Ampere–Maxwell law. [Based on your text]

***

#### 4. From Fields to Waves  
Because of displacement current:  
- A **changing electric field** produces a magnetic field.  
- A **changing magnetic field** produces an electric field.  

So electric and magnetic fields **keep creating each other** and can travel through space as:  
👉 **Electromagnetic (EM) waves.** [Based on your text]

***

#### 5. What Is an Electromagnetic Wave?  
An **electromagnetic wave** is a combination of:  
- An **electric field (E)**  
- A **magnetic field (B)**  
that **oscillate (vibrate)** and **move forward together**.

Important points:  
- **E and B are perpendicular to each other.**  
- **Both are perpendicular to the direction the wave is moving.**  
- The wave **does not need any material medium**; it can travel through vacuum. [Based on your text]

***

#### 6. Speed of EM Waves and Light  
From Maxwell’s equations, the speed of these waves in vacuum turned out to be:  
\[
c \approx 3 \times 10^8\ \text{m/s}
\]
This is the **same speed as light measured in optical experiments**.  
So Maxwell concluded:  
👉 **Light is an electromagnetic wave.** [Based on your text]

Other EM waves (radio, X‑rays, etc.) also travel at this same speed in vacuum.

***

#### 7. Where Do EM Waves Come From?  
EM waves are produced by **accelerated charges**.  
Examples:  
- A **charge oscillating up and down** (like in an antenna) produces EM waves.  
- The **frequency of the wave** is the **same as the frequency of oscillation** of the charge.  

Stationary charges and steady currents **do not** produce EM waves; only **accelerated (or oscillating) charges** do. [Based on your text]

***

#### 8. The Electromagnetic Spectrum  
All EM waves are the **same kind of wave**; they only differ in **wavelength** and **frequency**.  
The main types are (in order of increasing wavelength / decreasing frequency):  
1. **Gamma rays**  
2. **X‑rays**  
3. **Ultraviolet (UV)**  
4. **Visible light**  
5. **Infrared (IR)**  
6. **Microwaves**  
7. **Radio waves**  

There is **no sharp boundary** between one type and the next; the classification is mainly based on **how they are produced and detected**. [Based on your text]

***

#### 9. Quick Idea of Applications  
- **Radio waves**: used in radio, TV, mobile phones, and long‑distance communication.  
- **Microwaves**: used in radar, satellite communication, and microwave ovens.  
- **Infrared**: related to heat, used in remote controls, night‑vision, and thermal imaging.  
- **Visible light**: the part we see; it provides information about the world.  
- **Ultraviolet (UV)**: used for sterilisation, and can cause tanning or sunburn.  
- **X‑rays**: used in medical imaging and treatment.  
- **Gamma rays**: used in cancer treatment and come from nuclear processes.  

All these are just **different parts of the same electromagnetic spectrum**. [Based on your text]

***

#### 10. Key Takeaway (Simple)  
- **Displacement current** explains how a changing electric field makes a magnetic field.  
- This symmetry between electric and magnetic fields leads to **electromagnetic waves**.  
- **Light is one type** of EM wave; others include radio, microwaves, X‑rays, and gamma rays.  
- EM waves **travel through vacuum**, **carry energy**, and are **essential for modern technology and life on Earth**.



## Advanced Version (Electromagnetic Waves)

### 1. Introduction and Symmetry in Electromagnetism

From the basic laws of electromagnetism:

- Steady electric current produces a stationary magnetic field.  
- A magnetic field changing with time gives rise to an electric field (Faraday’s law of induction).  

James Clerk Maxwell argued that the converse must also hold:  
👉 **A time‑varying electric field produces a magnetic field.**

This completes the symmetry between electric and magnetic phenomena and leads naturally to the prediction of **self‑sustaining electromagnetic waves** propagating through space.

***

### 2. Displacement Current and Ampere–Maxwell Law

Consider a capacitor being charged with a time‑dependent current \( i(t) \).  
Using Ampere’s circuital law \( \oint \vec{B} \cdot d\vec{l} = \mu_0 i \) on a loop around the wire, we get a non‑zero magnetic field at a point P outside the capacitor.

However, if we choose a **pot‑like or tiffin‑box surface** with the same boundary loop, but with its bottom between the plates, **no conduction current passes through this surface**.

So on this surface the right‑hand side of Ampere’s law gives zero, yet the left‑hand side is the same. This is a **contradiction**.

Maxwell noticed that, while the conduction current is zero between the plates, the **electric field** between the plates is changing.

For a parallel‑plate capacitor of area \( A \) and charge \( Q \), the electric field between the plates is  
\[
E = \frac{Q}{\varepsilon_0 A} \quad \Rightarrow \quad \Phi_E = EA = \frac{Q}{\varepsilon_0}.
\]
Differentiating,
\[
\frac{d\Phi_E}{dt} = \frac{dQ}{dt} \cdot \frac{1}{\varepsilon_0} = \frac{i}{\varepsilon_0}.
\]
From this, Maxwell inferred that we must add a **displacement current**  
\[
i_d = \varepsilon_0 \frac{d\Phi_E}{dt},
\]
such that the **total current** through any surface bounded by the loop is the same, regardless of the surface chosen.
Thus the **total current** is  
\[
i = i_c + i_d,
\]
where
- \( i_c \): **conduction current** (flow of real charges in conductors),  
- \( i_d \): **displacement current** (due to time‑varying electric field / electric flux).

The **generalised Ampere–Maxwell law** then becomes  
\[
\oint \vec{B} \cdot d\vec{l} = \mu_0 i_c + \mu_0 \varepsilon_0 \frac{d\Phi_E}{dt}.
\]
This ensures that the predicted magnetic field is **consistent** for any surface with the same boundary loop.
***

### 3. Maxwell’s Equations in Vacuum

Maxwell collected four integral‑form equations that describe electric and magnetic fields and their sources (charge and current densities). They are often written in vacuum as:

1. **Gauss’s law for electricity**  
   \[
   \oint \vec{E} \cdot d\vec{A} = \frac{Q}{\varepsilon_0}
   \]
   (Electric flux through a closed surface is proportional to the net charge enclosed.)

2. **Gauss’s law for magnetism**  
   \[
   \oint \vec{B} \cdot d\vec{A} = 0
   \]
   (There is no net magnetic flux through any closed surface; no isolated magnetic monopoles.)

3. **Faraday’s law of induction**  
   \[
   \oint \vec{E} \cdot d\vec{l} = -\frac{d\Phi_B}{dt}
   \]
   (A time‑varying magnetic flux induces an electric field.)

4. **Ampere–Maxwell law**  
   \[
   \oint \vec{B} \cdot d\vec{l} = \mu_0 i_c + \mu_0 \varepsilon_0 \frac{d\Phi_E}{dt}
   \]
   (Magnetic field is produced by conduction current plus displacement current.)

These four equations, together with the **Lorentz force law** \( \vec{F} = q(\vec{E} + \vec{v} \times \vec{B}) \), mathematically express all basic laws of classical electromagnetism.
***

### 4. Electromagnetic Waves: Generation and Wave Equation

Maxwell’s key prediction: **Coupled, time‑varying electric and magnetic fields can propagate as waves through space.**  
These are **electromagnetic waves**—self‑sustaining oscillations of \( \vec{E} \) and \( \vec{B} \).

Important results:

- **Sources of EM waves:**  
  - **Stationary charges** produce only **electrostatic fields**.  
  - **Steady currents** produce **static magnetic fields**.  
  - **Accelerated charges** radiate **electromagnetic waves**.

  An **oscillating charge** (e.g., harmonic motion) acts as a **basic source** of EM waves. The frequency of the wave is the same as the oscillation frequency of the charge.

- **Hertz’s experiment (1887):**  
  Hertz produced and detected **radio‑like electromagnetic waves** in vacuum, confirmed their wavelength and speed, and showed that their reflection, refraction, and polarization resemble those of light and heat waves. This experiment **verified Maxwell’s theoretical prediction**.

- **Later work:**  
  - J. C. Bose produced **shorter‑wavelength waves** (microwaves) in the laboratory.  
  - Guglielmo Marconi transmitted EM waves **over long distances**, laying the foundation for **wireless communication**.
***

### 5. Mathematical Form of Plane EM Waves

Consider a **plane electromagnetic wave** propagating in vacuum along the **z‑direction**. The electric field \( \vec{E} \) and magnetic field \( \vec{B} \) are perpendicular to each other and to the direction of propagation.
We can write:
\[
E_x = E_0 \sin(kz - \omega t), \qquad
B_y = B_0 \sin(kz - \omega t),
\]
where:
- \( k = \frac{2\pi}{\lambda} \) is the **wave number**,  
- \( \omega \) is the **angular frequency**,  
- \( \omega/k = c \) is the **wave speed** in vacuum.
From Maxwell’s equations, one can derive the wave equation and show that  
\[
c = \frac{1}{\sqrt{\mu_0 \varepsilon_0}} \approx 3 \times 10^8\ \text{m/s}.
\]
This value turns out to be **very close to the measured speed of light**, suggesting that **light is an electromagnetic wave**.

The magnitudes of \( E \) and \( B \) at any point are related by  
\[
E_0 = c B_0,
\]
so  
\[
B_0 = \frac{E_0}{c}.
\]
This relation is used in example‑type calculations, such as:
- Given \( E = 6.3\ \text{V/m} \), find \( B \):  
  \[
  B = \frac{6.3}{3 \times 10^8} = 2.1 \times 10^{-8}\ \text{T}.
  \]
- Given the direction of \( \vec{E} \) and propagation, determine the direction of \( \vec{B} \) using \( \vec{E} \times \vec{B} \) along the propagation direction.
***

### 6. Nature, Speed, and Energy of EM Waves

- **Transverse nature:**  
  In vacuum, \( \vec{E} \) and \( \vec{B} \) are:
  - **Perpendicular to each other**,
  - **Perpendicular to the direction of propagation**.

- **No medium required:**  
  EM waves can propagate through **vacuum** (or free space) because the fields themselves sustain the oscillation.
- **Speed in vacuum:**  
  \[
  c = \frac{1}{\sqrt{\mu_0 \varepsilon_0}} \approx 3 \times 10^8\ \text{m/s}.
  \]
  The constancy of \( c \) with wavelength is experimentally well established and is used to **define the standard of length**.

- **Speed in a material medium:**  
  If the medium has permittivity \( \varepsilon \) and permeability \( \mu \), the wave speed becomes  
  \[
  v = \frac{1}{\sqrt{\mu \varepsilon}}.
  \]
  This leads to the concept of **refractive index** \( n = c/v \), which is introduced in the next chapter.

- **Energy transport:**  
  EM waves **carry energy**; the energy comes at the expense of the energy of the **accelerated charge** that radiates.  
  The **average energy density** of the electric field equals the **average energy density** of the magnetic field for a harmonic plane wave:
  \[
  u_E = \frac{1}{2} \varepsilon_0 E_0^2, \quad
  u_B = \frac{B_0^2}{2\mu_0}, \quad
  \text{and} \quad
  u_E = u_B.
  \]
  This is explored in exercise‑style problems (e.g., average energy‑density equality at \( 2.0 \times 10^{10}\ \text{Hz} \), \( E_0 = 48\ \text{V/m} \)).
***

### 7. Electromagnetic Spectrum (Detailed View)

The spectrum of EM waves in principle spans an **infinite** range of wavelengths, from **gamma rays** to **long radio waves**. The classification is mainly based on **how they are produced and detected**, not on sharp boundaries.

#### Broad wavelength/frequency regions

- **Gamma rays:**  
  \(\lambda \sim 10^{-14}\ \text{m}\) to below \(10^{-12}\ \text{m}\); produced in **nuclear reactions** and by **radioactive nuclei**; used to destroy cancer cells.

- **X‑rays:**  
  \(\lambda \sim 10^{-10}\ \text{m}\) down to \(\sim 10^{-13}\ \text{m}\); produced by **high‑energy electrons** hitting a metal target; used in diagnostics and cancer treatment.
- **Ultraviolet (UV):**  
  \(\lambda \sim 4 \times 10^{-7}\ \text{m}\) to \(6 \times 10^{-10}\ \text{m}\); produced by **very hot bodies** and **special lamps**; the Sun is a major source, but most UV is absorbed in the **ozone layer**.
- **Visible light:**  
  \(\lambda \sim 700\ \text{nm}\) to \(400\ \text{nm}\); detected by the **human eye**; the Sun’s peak radiation is in this region, which matches the **evolutionary tuning** of human vision.

- **Infrared (IR):**  
  Just below visible; produced by **hot bodies** and **molecules**; often called **heat waves** because many molecules (e.g., H₂O, CO₂, NH₃) absorb it and increase their thermal energy.

- **Microwaves:**  
  Short‑wavelength radio waves, \( f \sim \text{GHz} \); produced by **klystrons, magnetrons, Gunn diodes**; used in radar, speed‑guns, and microwave ovens (where frequency is tuned to water‑molecule resonance).

- **Radio waves:**  
  Longest wavelength EM waves, up to \(\lambda \sim 10^6\ \text{m}\); produced by **accelerated electrons in conducting wires / antennas**; used in **radio, TV, FM, AM, mobile communication, and satellite systems**.

#### Photon‑energy perspective (optional but relevant)

Using the **photon energy** formula \( E = h\nu \) (from quantum ideas), one can compute the energy (in eV) across the spectrum.  
This exercise shows:
- **High‑frequency** waves (gamma, X‑rays, UV) carry **high‑energy photons**, which can ionize atoms and damage tissue.  
- **Low‑frequency** waves (radio, microwaves, IR) carry **low‑energy photons**, which mainly cause heating or collective motion of charges and molecules.
***

### 8. Interaction with Matter and System Size

The way EM waves interact with matter depends strongly on **frequency** (or wavelength) and on the **nature of atoms and molecules** in the medium.
Key qualitative ideas:

- **Infrared waves:**  
  Vibrations of **entire atoms or molecules** are excited; this increases internal energy and temperature, hence the term **heat waves**.

- **Visible and UV:**  
  Cause **electronic transitions** (jump between energy levels) in atoms. UV can also **ionize** atoms and is strongly absorbed by glass and ozone.
- **X‑rays:**  
  Interact with **inner‑shell electrons** and can penetrate soft tissue but are absorbed by bone, making them useful for imaging.

- **Radio and microwaves:**  
  Typically interact via **collective oscillation of free or bound charges** in conductors and dielectrics.
A useful rule of thumb:
- The **wavelength** of EM radiation is often comparable to the **characteristic size** of the radiating system.  
  - **Gamma rays** (nuclear size) \(\sim 10^{-14}\ \text{m}\).  
  - **X‑rays** from heavy atoms.  
  - **Radio waves** from antennas roughly of the order of the wavelength.  
  - **Visible light** from atoms, but wavelength is **much larger** than the atom size.
***

### 9. Key Conclusions and Summary of Physics

- **Displacement current** \( i_d = \varepsilon_0 d\Phi_E/dt \) resolves the inconsistency in Ampere’s law and ensures that the **total current** is well‑defined for any surface with a given boundary.
- **Maxwell’s equations** predict that:
  - **Time‑varying electric and magnetic fields** can mutually sustain each other.  
  - The resulting **electromagnetic waves** propagate in vacuum at speed \( c = 1/\sqrt{\mu_0 \varepsilon_0} \).  
  - **Light** is one such electromagnetic wave.

- **Accelerated charges** (especially oscillating charges) are the **fundamental sources** of EM radiation; the frequency of the wave equals the oscillation frequency of the source.

- The **entire electromagnetic spectrum**—from gamma rays to radio waves—consists of the same underlying physical phenomenon, differing only in **wavelength/frequency** and hence in **ways of production, detection, and interaction with matter**.

- EM waves **carry energy and momentum**, enabling:
  - **Radio and TV broadcasting**,  
  - **Microwave communication and radar**,  
  - **Medical imaging and therapy**,  
  - **Solar energy transfer** to Earth.

